KEEP/services/redis

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+DBDIR=/var/lib/redis
+
+if ! grep -q '^redis:' /etc/passwd ; then
+        busybox addgroup -S redis
+        busybox adduser -h "$DBDIR" -s /bin/nologin -S -D -H -G redis redis
+	chown redis:redis /etc/redis.conf
+fi
+
+if [ ! -d "$DBDIR" ] ; then
+        install -D -d -m 755 "$DBDIR"
+        chown -R redis "$DBDIR"
+        chgrp -R redis "$DBDIR"
+fi
+
+
+
+[ "$1" = "--prereqs" ] && exit 0
+
+
+# http://redis.io/topics/latency
+echo 'never' > /sys/kernel/mm/transparent_hugepage/enabled
+
+# see tcp-backlog in redis.conf
+echo 511 > /proc/sys/net/core/somaxconn
+
+su -s /bin/sh -c '/bin/redis-server /etc/redis.conf' - redis
```

Comment:
I didn't know `chpst` and it seems to be awesome :+1: , it can even do the chroot in the same command.

How about if I modify the package to work in a chroot by default? Or should I put this in another service file/a wiki entry?